src/modules/users/Config.cpp

```diff
@@ -279,6 +284,30 @@ guessProductName()
     }
     return dmiProduct;
 }
+#ifdef HAVE_ICU
+static QString
+transliterate( const QString& input )
+{
+    static UErrorCode ue = UErrorCode::U_ZERO_ERROR;
+    static auto transliterator = std::unique_ptr< icu::Transliterator >(
+                icu::Transliterator::createInstance( "Any-Latin; Latin-ASCII", UTRANS_FORWARD, ue )
+                );
+
+    if(ue!=0){
+        cWarning() << "Can't create transliterator";
+
+        //it'll be checked later for non-ASCII characters
+        return input;
+    }
+
+    auto transliterable = icu::UnicodeString( input.toUtf8().data() );
```

Comment:
Please construct the `icu::UnicodeString` directly from UTF-16 data rather than roundtripping through UTF-8. Using `input.utf16()` instead of `input.toUtf8().data()` should do the trick.

Reply:
Thanks